tidb-binlog/driver/reader/reader.go

```diff
@@ -90,15 +90,20 @@ func NewReader(cfg *Config) (r *Reader, err error) {
 		clusterID: cfg.ClusterID,
 	}
 
-	r.client, err = sarama.NewClient(r.cfg.KafkaAddr, nil)
+	conf := sarama.NewConfig()
+	// set to 10 minutes to prevent i/o timeout when reading huge message
+	conf.Net.ReadTimeout = KafkaReadTimeout
+	conf.Consumer.Return.Errors = true
```

Comment:
set false to make it just log for error, because it don't handle Error() now.
 
in the seeker, you can still set true